catalog/core/catalog-core-migratable/src/main/java/org/codice/ddf/catalog/migratable/impl/CatalogMigratableImpl.java

```diff
@@ -132,6 +133,11 @@ public MigrationMetadata export(Path exportPath) throws MigrationException {
         return new MigrationMetadata(warnings);
     }
 
+    @Override
+    public String getDescription() {
```

Comment:
Iff you really want `getDescription` to be in the subclasses and not the abstract base, then it's no longer `@Override`.